src/llm/apis/openai_completions.cpp

```diff
@@ -454,7 +448,9 @@ std::string OpenAIChatCompletionsHandler::serializeUnaryResponse(const std::vect
                     // top_logprobs are currently hardcoded to return only one logprob to comply with the API
                     // for full support significant changes on GenAI side are required
```

Comment:
With change below this comment is no longer valid, please adjust it to inform it's always empty array now